qa-ovh-maintenance/update_static_slaves.groovy

```diff
@@ -61,8 +105,10 @@ def update_static_slaves(boolean doConfirm=false) {
       }
     }
   }
```

Comment:
~here, the updated slaves need to be put online back~
the start Shell recreates the slave, so it is online by default, the default one being actually deleted.